config/environments/development.rb

```diff
@@ -9,13 +9,28 @@
   # Do not eager load code on boot.
```

Comment:
Most of these environments file diffs are the result of rails 5. Most of these options no longer are needed. 

You can do a git blame/history look up on these generated files if your curious on the why behind them here:
https://github.com/rails/rails/blob/master/railties/lib/rails/generators/rails/app/templates/config/environments/development.rb.tt